activerecord/CHANGELOG.md

```diff
@@ -425,6 +431,5 @@
     Previously this method always returned an empty array.
 
     *Kevin McPhillips*
-
-
+    
```

Comment:
Can you undo the whitespace changes here, and squash your commits again? After that, this looks good to me - thanks for being so responsive to feedback @kinnrot! 🙏

Reply:
is that ok?

Reply:
There's still a diff here, but I'm not sure if it's a blocker for merging - maybe @rafaelfranca can let us know.

Reply:
This diff can easily be removed by deleting the four extra whitespaces and adding one blank line.